### PR TITLE
Add `offline-url` documentation

### PR DESCRIPTION
**Pull request by GabrielEValenzuela**
# Objective
This PR aims to address the omission of the `offline-url` option in the existing documentation for the Vulnerability Detection module. By documenting the valid values for the `offline-url` field, we provide users with comprehensive guidance on how to configure this feature for offline updates, enhancing usability and functionality of the system.

## Description
- Feedback and user inquiries revealed a gap in our documentation regarding the configuration of the `offline-url` option in the Vulnerability Detection module. This option is critical for environments where direct internet access is restricted or unavailable.
- To remedy this, we have updated the documentation to include a dedicated section for the `offline-url` option. This section details the purpose of the option, describes its use cases, and lists all valid values and formats that can be used to configure the system for offline operation.
- **Key Additions**:
  - **Purpose and Usage**: Explanation of why and when to use the `offline-url` option.
  - **Valid Values**: Comprehensive list of valid URL formats and configurations supported by the system, such as local file paths or network-shared folders, with examples for each.

### CODE REVIEW

File: source/user-manual/reference/ossec-conf/vuln-detector.rst [hunk 1]
```diff
@@ -32,6 +32,10 @@ Options
 +---------------------------+------------------------------------+
 | `index-status`_           | ``yes``, ``no``                    |
 +---------------------------+------------------------------------+
+| `offline-url`_            | ``<path>`` (starting with file://) |
+|                           | or ``<URL>``                       |
+|                           | (starting with http:// or https://)|
```
Comment (MiguelazoDS) [inline source/user-manual/reference/ossec-conf/vuln-detector.rst:37]: I would remove the hyperlinks here. 

Here's an example I found within the documentation 

https://documentation.wazuh.com/current/user-manual/user-administration/single-sign-on/administrator/pingone.html
image

File: source/user-manual/reference/ossec-conf/vuln-detector.rst [hunk 2]
```diff
@@ -66,6 +70,20 @@ Enables indexing of vulnerability inventory data.
...
+
++--------------------+-----------------------------+
+| **Default**        | Empty                       |
++--------------------+-----------------------------+
+| **Allowed values** | ``<path>``, ``<URL>``       |
++--------------------+-----------------------------+
+
+``<path>``: A local file path starting with file://, pointing to offline content stored on the device.
+
+``<URL>``: A URL starting with either http:// or https://, pointing to online content accessible via the internet or local network.
```
Comment (MiguelazoDS) [inline source/user-manual/reference/ossec-conf/vuln-detector.rst:85]: I would remove the hyperlinks here.

**Review by MiguelazoDS — APPROVED** [review]
GJ! no changes required just a comment to discuss, probably is not relevant.

File: source/user-manual/reference/ossec-conf/vuln-detector.rst (hunk 1 above)
Comment (GabrielEValenzuela) [reply source/user-manual/reference/ossec-conf/vuln-detector.rst]: Done. Thx!

File: source/user-manual/reference/ossec-conf/vuln-detector.rst (hunk 2 above)
Comment (GabrielEValenzuela) [reply source/user-manual/reference/ossec-conf/vuln-detector.rst]: Done. Thx!

**Review by MiguelazoDS — APPROVED** [review]

**Review by sebasfalcone — APPROVED** [review]
LGTM

File: source/user-manual/reference/ossec-conf/vuln-detector.rst [hunk 3]
```diff
@@ -66,6 +70,20 @@ Enables indexing of vulnerability inventory data.
 | **Allowed values** | ``yes``, ``no``             |
 +--------------------+-----------------------------+
 
+offline-url
+^^^^^^^^^^^
+Allows to specify a path or URL for offline content access.
```
Comment (javimed) [inline source/user-manual/reference/ossec-conf/vuln-detector.rst:75]:
```suggestion
File path or URL for offline content access.
```

File: source/user-manual/reference/ossec-conf/vuln-detector.rst [hunk 4]
```diff
@@ -23,15 +23,19 @@ Options
...
++---------------------------+--------------------------------------+
+| Options                   | Allowed values                       |
++===========================+======================================+
+| `enabled`_                | ``yes``, ``no``                      |
++---------------------------+--------------------------------------+
+| `feed-update-interval`_   | Positive number + Time unit suffix   |
++---------------------------+--------------------------------------+
+| `index-status`_           | ``yes``, ``no``                      |
++---------------------------+--------------------------------------+
+| `offline-url`_            | ``<path>`` (starting with \file://)  |
```
Comment (javimed) [inline source/user-manual/reference/ossec-conf/vuln-detector.rst:35]: We could use `file://<LOCAL_FILE_PATH>` and `http[s]://<URL>` instead

File: source/user-manual/reference/ossec-conf/vuln-detector.rst [hunk 5]
```diff
@@ -66,6 +70,20 @@ Enables indexing of vulnerability inventory data.
...
 +--------------------+-----------------------------+
 
+offline-url
+^^^^^^^^^^^
+Allows to specify a path or URL for offline content access.
+
++--------------------+-----------------------------+
+| **Default**        | Empty                       |
++--------------------+-----------------------------+
+| **Allowed values** | ``<path>``, ``<URL>``       |
```
Comment (javimed) [inline source/user-manual/reference/ossec-conf/vuln-detector.rst:80]: We could use `file://<LOCAL_FILE_PATH>` and `http[s]://<URL>` instead

**Review by javimed — CHANGES_REQUESTED** [review]
FYI: Created issue https://github.com/wazuh/wazuh-documentation/issues/7207

File: source/user-manual/reference/ossec-conf/vuln-detector.rst [hunk 6]
```diff
@@ -66,6 +70,20 @@ Enables indexing of vulnerability inventory data.
...
++--------------------+-----------------------------+
+
+``<path>``: A local file path starting with \file://, pointing to offline content stored on the device.
+
+``<URL>``: A URL starting with either \http:// or \https://, pointing to online content accessible via the internet or local network.
```
Comment (javimed) [inline source/user-manual/reference/ossec-conf/vuln-detector.rst:85]: It couldn't be "via the internet" if we're referring to offline content.

**Review by javimed — CHANGES_REQUESTED** [review]

File: source/user-manual/reference/ossec-conf/vuln-detector.rst (hunk 6 above)
Comment (javimed) [inline source/user-manual/reference/ossec-conf/vuln-detector.rst:85]:
```suggestion
``<URL>``: A URL starting with either \http:// or \https://, pointing to local network content or online content accessible via the internet.
```

**Review by javimed — CHANGES_REQUESTED** [review]

File: source/user-manual/reference/ossec-conf/vuln-detector.rst [hunk 7]
```diff
@@ -66,6 +70,20 @@ Enables indexing of vulnerability inventory data.
 | **Allowed values** | ``yes``, ``no``             |
 +--------------------+-----------------------------+
 
+offline-url
+^^^^^^^^^^^
```
[the remark this answers is not in the document]
Comment (GabrielEValenzuela) [reply source/user-manual/reference/ossec-conf/vuln-detector.rst:72]: Done!

File: source/user-manual/reference/ossec-conf/vuln-detector.rst (hunk 3 above)
Comment (GabrielEValenzuela) [reply source/user-manual/reference/ossec-conf/vuln-detector.rst]: Done!

File: source/user-manual/reference/ossec-conf/vuln-detector.rst (hunk 4 above)
Comment (GabrielEValenzuela) [reply source/user-manual/reference/ossec-conf/vuln-detector.rst]: Good catch!

File: source/user-manual/reference/ossec-conf/vuln-detector.rst (hunk 5 above)
Comment (GabrielEValenzuela) [reply source/user-manual/reference/ossec-conf/vuln-detector.rst]: I agree!

File: source/user-manual/reference/ossec-conf/vuln-detector.rst (hunk 6 above)
Comment (GabrielEValenzuela) [reply source/user-manual/reference/ossec-conf/vuln-detector.rst]: Done!

**Review by javimed — DISMISSED** [review]

**Review by sebasfalcone — APPROVED** [review]
LGTM!

**Review by javimed — APPROVED** [review]

**Pull request merged**